src/components/ground-nav/ground-nav.scss

```diff
@@ -0,0 +1,197 @@
+@use '../../design-tokens/breakpoint.yml';
+@use '../../design-tokens/colors.yml';
+@use '../../design-tokens/font.yml';
+@use "../../design-tokens/sizes.yml";
+@use '../../mixins/ms';
+@use '../../mixins/unit';
+
+// Whitespace used for gaps, padding and margin
+// (It uses `rem` so space will be consistent throughout the component)
+$_ground-nav-space: ms.step(1, 1rem);
+// Negative value used to reduce the space around elements as well as position elements.
+$_ground-nav-negative-space: ($_ground-nav-space * -1);
+$_ground-nav-border-width: sizes.$edge-medium;
+$_ground-nav-border-style: solid;
+$_ground-nav-border-color: colors.$gray-light;
+
+.c-ground-nav__action,
+.c-ground-nav__content {
+  padding: $_ground-nav-space;
+}
+
+.c-ground-nav__action-inner {
+  position: relative;
+
+  @media (min-width: breakpoint.$l) {
+    align-items: center;
+    display: flex;
+  }
+}
+
+/**
+ * 1. Creates space between the action lead in and action title.
+ * 2. Adds horizontal space when these elements are side by side.
+ */
+.c-ground-nav__action-title {
+  margin-top: $_ground-nav-space; /* 1 */
+
+  @media (min-width: breakpoint.$l) {
+    margin: ms.step(4) $_ground-nav-space; /* 2 */
+  }
+}
+
+/**
+ * The illustration is aligned with the footer content.
+ * As the screen size increases, we decrease the size
+ * of the illustration so that it doesn't get too large.
+ */
+
+.c-ground-nav__action-illustration {
+  bottom: $_ground-nav-negative-space;
+  position: absolute;
+  right: $_ground-nav-negative-space;
+  width: 40%;
+
+  @media (min-width: breakpoint.$m) {
+    right: 0;
+    width: calc(100% / 3);
+  }
+
+  @media (min-width: breakpoint.$l) {
+    width: 25%;
+  }
+}
+
+.c-ground-nav__content {
+  background-color: colors.$gray-lighter;
+}
+
+/**
+ * Creates vertical space between elements when grid isn't supported.
+ */
+.c-ground-nav__content-inner > * + * {
+  margin-top: $_ground-nav-space;
+}
+
+/**
+ * 1. Removes `margin-top` so the organization information is
+ *    aligned with the other footer elements
+ */
+.c-ground-nav__organization {
+  font-style: normal;
```

Comment:
Is this here because of the default styles for `address`?

Do you think we should set this for `address` by default? I find it had to imagine we'll want an address that is italicized by default, but I'd love a second opinion. 🤔

Reply:
Yes, I can update the default style for `address` instead. We can always change it in the future if we need to.